.kitchen.yml

```diff
@@ -19,16 +19,30 @@ provisioner:
     base:
       '*':
         - extenda-repo
-  pillars:
-    top.sls:
-      base:
-        "*":
-          - vault
   grains:
-    customer: test-customer
+    customer: auc
```

Comment:
@rawkode Why are you changing the default customer to "Auc" instead of "test-customer"?

Reply:
This was a change we made during a call, for testing, that shouldn't be here. Updating now :+1: